### PR TITLE
Improvements and small corrections to migration guide

### PR DESCRIPTION
**Pull request by marcospereira**
## Purpose

1. Add Netty 4.1 upgrade information
2. Correct some format details
3. Reword a phrase of Akka migration.

### CODE REVIEW

File: documentation/manual/releases/release26/migration26/Migration26.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ libraryDependencies += "com.typesafe.play" %% "play-iteratees-reactive-streams"
 
 Finally, Play Iteratees has a separate versioning scheme, so the version no longer is in sync with the Play version.
 
-## Writeable[JsValue] changes
+## `Writeable[JsValue] changes
```
Comment (gmethvin) [inline documentation/manual/releases/release26/migration26/Migration26.md]: missing ` at the end?

Comment (marcospereira) [inline documentation/manual/releases/release26/migration26/Migration26.md]: Fixed. :-)

File: documentation/manual/releases/release26/migration26/Migration26.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<!--- Copyright (C) 2009-2017 Lightbend Inc. <https://www.lightbend.com> -->
+<!--- iopyright (C) 2009-2017 Lightbbleend Inc. <https://www.lightbend.com> -->
```
Comment (PromanSEW) [inline documentation/manual/releases/release26/migration26/Migration26.md]: Why?

Comment (marcospereira) [inline documentation/manual/releases/release26/migration26/Migration26.md]: Code catch. I don't know what happened here.

**Pull request merged**